### ISSUE TITLE
Issue with embedded MakeCode environments in OneNote 

### ISSUE THREAD

**Issue by kiki-lee** [microsoft/pxt-microbit#4828]
From a Twitter user:

> _there's this issue with embedded makecode environments at a certain resolution where the emulator doesn't load. any chance theres a fix comming soon? Thanks_

![image](https://user-images.githubusercontent.com/1633958/185248446-eaaea12c-ed55-43b2-8a9b-e341fccb942b.png)

When asked to try the same thing on the desktop version:

> _same result._

![image](https://user-images.githubusercontent.com/1633958/185248417-7c4467c3-c212-4110-928a-278dc39e67e9.png)

**Comment by Jaqster** [comment]
I can repro this.  And it's across editors...
![image](https://user-images.githubusercontent.com/22108954/185474893-e18e2c45-34ce-490a-a41c-4cc2213527ce.png)

**Comment by abchatra** [comment]
This is likely because we dropped support for IE11. If I remember correctly this was dependent on IE11 rendering engine. Do we have contact in the team? We should check if there is a way to invoke modern engine .

**Comment by abchatra** [comment]
I got the contact. Lets first investigate and ensure that is the issue.

**Comment by thsparks** [comment]
I don't think this is because of IE. When I inspect the element in OneNote and view console output, we can see it says it's using Chrome (which probably means it's actually edge): 
<img width="140" alt="image" src="https://github.com/microsoft/pxt-microbit/assets/69657545/a8b3a41f-66f6-49b8-a69d-9118e945ab6c">

**Comment by thsparks** [comment]
This bug repros even on just a regular share page. I think the reason we noticed it in OneNote is because it only happens on small sizes, and the default OneNote embed section is pretty tiny.

Still investigating root cause...

**Comment by abchatra** [comment]
Awesome find

**Comment by thsparks** [comment]
This is partially addressed by https://github.com/microsoft/pxt/pull/9570 and should at least be usable once that gets released. That said, there may be future improvements worth considering here, like:
1. Fixing the gap on the right-hand side where background doesn't fully extend to the edge.
2. Bring back the sim toolbar, but probably remove "debug" from it.
3. Make the download bar blend a bit more seamlessly into the page, or remove it.

Numbers 2 and 3 may need some design discussion with the team.

**Comment by thsparks** [comment]
@abchatra Let me know if you want to include any of those additional changes mentioned above in the hotfix, or if the partial fix is sufficient. Personally, I'd be inclined to leave them for a future release.

**Comment by abchatra** [comment]
Partial fix is sufficient. Lets hotfix it in microbit as well as arcade

**Comment by jwunderl** [comment]
Removing milestone as it's fixed, but leaving issue as there are follow up improvements we want to do remaining

**Comment by abchatra** [comment]
Lets close this and open new issue for other pending things if it is important.
